### ISSUE TITLE
React Native import of SendBirdCall Throws Unhandled Promise Rejection Error

### ISSUE THREAD

**Issue by dquessenberry** [sendbird/sendbird-calls-javascript#1]
I am attempting to setup SendBird calls in a React Native application and the import statement, `import SendBirdCall from 'sendbird-calls';` throws the following error: `Unhandled promise rejection Error: An internal error has occurred, please try again.`. 

The app already has SendBird messaging installed and working properly. I am unable to initialize or run any SendBirdCall listeners, auth, etc. functions, as the import statement itself seems to be throwing an error. 

In order to replicate this issue, initialize a new React Native application or fire up an existing React Native application. Then install the SendBird Call module, i.e. `yarn add sendbird-calls` and add the Sendbird Calls import statement to the App.js. Then run the app and you should see the error.

**App Information:**
"react": "16.8.6",
"react-native": "0.60.5",
"sendbird": "^3.0.108",
"sendbird-calls": "^1.0.1",

<img width="478" alt="Screen Shot 2020-04-11 at 10 46 20 AM" src="https://user-images.githubusercontent.com/10273674/79050876-d1722c00-7be1-11ea-9b07-bd684433f348.png">
<img width="478" alt="Screen Shot 2020-04-11 at 10 46 30 AM" src="https://user-images.githubusercontent.com/10273674/79050892-d6cf7680-7be1-11ea-8d38-fe3db204f515.png">
<img width="478" alt="Screen Shot 2020-04-11 at 10 46 37 AM" src="https://user-images.githubusercontent.com/10273674/79050902-dc2cc100-7be1-11ea-8b1a-150e0aaebf82.png">
<img width="478" alt="Screen Shot 2020-04-11 at 10 46 49 AM" src="https://user-images.githubusercontent.com/10273674/79050904-e222a200-7be1-11ea-9fdb-a1978df01aff.png">

**Comment by jungdj** [comment]
Hi dquessenberry, I'm Cobb from Sendbird Javascript SDK team, and thanks for reporting this issue.
 However as our Sendbird call service is currently not supporting React Native officially,  there might be some problems related to it. We are tracking on this issue but we cannot promise any timeline on it.
Thanks

**Comment by jungdj** [comment]
Hi dquessenberry

I tried to reproduce it but couldn't. Did you try to initialize and authenticate SendBirdCalls after importing it? Does the same error happens even it's not initialized?

Thanks

**Comment by jungdj** [comment]
Closing this issue due to reproducing unavailable.
If you're still encountering this issue please re-open.

**Comment by dquessenberry** [comment]
Hi @jungdj, I just wanted to follow up here and make note of my issue. I was able to "beautify" the SendBirdCall minified js and throw in some console log statement to get more information of what is going one, and it appears the promise rejection error I was receiving is related to signing in anonymously with firebase at the start of my application. I don't remember why my app was signing in anonymously with firebase to begin with, but once I commented out that code the SendBirdCalls import was no longer throwing the promise rejection error. Thanks!
